IyiKuyoro/portfolio-react
# #170080942: Fix Edit Article Issue

#### What does this PR do?

This PR fixes the issue with editing an article when there is one saved in the draft

#### Description of Task to be completed?

- [x] refactor edit article compoenent
- [x] refactor logout reducer
- [x] edit article e2e test

#### How should this be manually tested?

- Log in and write some content on the write article page. Wait for it to save to the db
- Navigate to a published article and click the edit button. The article should load regardless of the saved draft.

#### What are the relevant pivotal tracker stories?

#170080942

#### Any background context you want to add?

N/A

#### Screenshots

N/A
